src/mdast2jcr/partials/link.js

```diff
@@ -16,7 +16,7 @@ import Handlebars from 'handlebars';
 function link(context) {
   const uniqueName = Handlebars.helpers.nameHelper.call(context, 'button');
   const { url, title, children: [child] } = context;
-  return `<button${uniqueName} sling:resourceType="core/franklin/components/button/v1/button" jcr:primaryType="nt:unstructured" link="${url}" linkTitle="${title}" linkText="${child.value}" />\n`;
+  return `<button${uniqueName} sling:resourceType="core/franklin/components/button/v1/button" jcr:primaryType="nt:unstructured" link="${url}" linkTitle="${title !== null ? title : ''}" linkText="${child.value}" />\n`;
```

Comment:
Remove \n.

Reply:
Question for you.  What should the rule be for properties that do not have a value?  Should we still add that property to the jcr node and set an empty value for it?  Or should we not create the property?

Reply:
Both should be fine. But I think removing the attribute at is more clean, will remove it.

Reply:
fixed, for all link, strong and emphasis